build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
   ext.junitJupiterVersion = '5.6.1'
-  ext.gmaVersion = '0.2.80'
+  ext.gmaVersion = '0.3.0'
```

Comment:
after making the versioning change I suggested in datahub-gma, we'll want this to be more like `0.2.81` 👍